dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathChecker.java

```diff
@@ -108,14 +118,22 @@ public static void main(String[] arguments)
   public ClasspathCheckReport findLinkageErrors() throws IOException {
     ImmutableList<Path> jarPaths = classDumper.getInputClassPath();
 
-    ImmutableMap.Builder<Path, SymbolReferenceSet> jarToSymbols = ImmutableMap.builder();
+    ImmutableMap.Builder<Path, SymbolReferenceSet> jarToSymbolBuilder = ImmutableMap.builder();
     for (Path jarPath : jarPaths) {
-      jarToSymbols.put(jarPath, ClassDumper.scanSymbolReferencesInJar(jarPath));
+      jarToSymbolBuilder.put(jarPath, ClassDumper.scanSymbolReferencesInJar(jarPath));
     }
+    ImmutableMap<Path, SymbolReferenceSet> jarToSymbols = jarToSymbolBuilder.build();
+
+    ImmutableSet.Builder<ClassSymbolReference> classReferenceBuilder = ImmutableSet.builder();
+    for (SymbolReferenceSet symbolReferenceSet : jarToSymbols.values()) {
+      classReferenceBuilder.addAll(symbolReferenceSet.getClassReferences());
+    }
+    this.classSymbolGraph = ClassSymbolGraph.create(classReferenceBuilder.build()
+        , entryPoints);
```

Comment:
The graph can be built here from `SymbolReferenceSet`, making it difficult to make it as `final` instance member.